pxblue/react-component-library
# Changed import statements of material-ui components to default imports

Fixes #321

#### Changes proposed in this Pull Request:
- Changed import statements of material-ui components to default imports
- Comp props and other things than components still be imported as named imports


#### To Test:
1. Start a new React project with the pxblue cli via command line prompt npx -p @pxblue/cli pxb new (https://pxblue.github.io/development/cli) 
2. Run yarn build from the root of the react-component-library in the updated branch you have locally.
3. The previous command should create a dist folder... copy the contents of that dist folder into your new cli generated projects node_modules/@pxblue/react-native-components folder.
4. Add the source-map-explorer to the cli generated project as a dev dependency and add "analyze" script "analyze": "source-map-explorer 'build/static/js/*.js'" 
5. Run yarn build in the new cli project
6. Run yarn analyze
7. Compare the new chart to the one posted in the PR
